orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/config/EchoConfiguration.groovy

```diff
@@ -106,4 +108,9 @@ class EchoConfiguration {
   ApplicationListener<ExecutionEvent> echoNotifyingPipelineExecutionListenerAdapter(EchoNotifyingExecutionListener echoNotifyingExecutionListener, ExecutionRepository repository) {
     return new ExecutionListenerAdapter(echoNotifyingExecutionListener, repository)
   }
+
+  @PostConstruct
+  void registerTriggers() {
+    objectMapper.registerSubtypes(CronTrigger)
+  }
```

Comment:
Why do we need this if we've got this? https://github.com/spinnaker/orca/pull/1890/files#diff-9f145d81571e753ab25e9263c87a4ffdR37